Taskfile.dist.yaml

```diff
@@ -35,3 +280,427 @@ tasks:
+        s3_access_key_id={{.s3_access_key}} s3_bucket_name=trdl \
+        s3_region=ru-central1 s3_endpoint=http://$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        required_number_of_verified_signatures_on_commit=0 \
+        git_repo_url=https://github.com/werf/trdl
+      - |
+        VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.vault_port}} \
+        vault secrets enable -path=kubedog vault-plugin-secrets-trdl
+        VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.vault_port}} \
+        vault write kubedog/configure s3_secret_access_key={{.s3_secret_key}} \
+        s3_access_key_id={{.s3_access_key}} s3_bucket_name=kubedog \
+        s3_region=ru-central1 s3_endpoint=http://$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        required_number_of_verified_signatures_on_commit=0 \
+        git_repo_url=https://github.com/werf/kubedog
+      - |
+        VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.vault_port}} \
+        vault secrets enable -path=werf vault-plugin-secrets-trdl
+        VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.vault_port}} \
+        vault write werf/configure s3_secret_access_key={{.s3_secret_key}} \
+        s3_access_key_id={{.s3_access_key}} s3_bucket_name=werf \
+        s3_region=ru-central1 s3_endpoint=http://$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        required_number_of_verified_signatures_on_commit=0 \
+        git_repo_url=https://github.com/werf/werf \
+        git_trdl_channels_branch=multiwerf
+    vars:
+      dir: '{{.dir | default "server"}}'
+
+  vault:logs:
+    desc: Get logs from vault container
+    cmds:
+      - docker logs -f {{.vault_container}}
+
+  clean:
+    desc: Cleanup development environment
+    dir: "{{.dir}}"
+    cmds:
+      - rm -f ./vault/plugins/vault-plugin-secrets-trdl
+      - docker rm -f {{.minio_container}} || true
+      - docker rm -f {{.vault_container}} || true
+      - docker run --rm --volume $(pwd):/wrk alpine rm -rf /wrk/{{.minio_data_dir}}
+    vars:
+      dir: '{{.dir | default "server"}}'
+
+  doc:gen:
+    desc: Regenerate references
+    dir: "{{.dir}}"
+    deps:
+      - doc:reference:cli
+      - doc:reference:vault-plugin
+      - doc:reference:sidebar
+    vars:
+      dir: '{{.dir | default "docs"}}'
+
+  doc:reference:cli:
+    desc: Regenerate CLI reference
+    dir: "{{.dir}}"
+    cmds:
+      - cd ../client && go install github.com/werf/trdl/client/cmd/trdl
```

Comment:
Build the binary via tasks that we defined previously, i. e. `build:client:...`.